### PR TITLE
Correction to Cerificate typo, missing `t`

### PR DESCRIPTION
**Pull request by heldersepu**
### Description
Just a small correction to `Cerificate` typo, missing `t` in Certificate

### CODE REVIEW

File: website/docs/cdktf/python/r/acm_certificate.html.markdown [hunk 1]
```diff
@@ -44,7 +44,7 @@ New certificate materials can be supplied to an existing imported certificate to
 
 ## Private Certificates
 
-Private certificates are issued by an ACM Private Cerificate Authority, which can be created using the resource type [`aws_acmpca_certificate_authority`](acmpca_certificate_authority.html).
+Private certificates are issued by an ACM Private Certificate Authority, which can be created using the resource type [`aws_acmpca_certificate_authority`](acmpca_certificate_authority.html).
```
Comment (breathingdust) [inline website/docs/cdktf/python/r/acm_certificate.html.markdown:47]: The language specific files are generated automatically from the main file in `/website/docs/r` so this change should be removed.

```suggestion
Private certificates are issued by an ACM Private Cerificate Authority, which can be created using the resource type [`aws_acmpca_certificate_authority`](acmpca_certificate_authority.html).
```

File: website/docs/cdktf/typescript/r/acm_certificate.html.markdown [hunk 2]
```diff
@@ -44,7 +44,7 @@ New certificate materials can be supplied to an existing imported certificate to
 
 ## Private Certificates
 
-Private certificates are issued by an ACM Private Cerificate Authority, which can be created using the resource type [`aws_acmpca_certificate_authority`](acmpca_certificate_authority.html).
+Private certificates are issued by an ACM Private Certificate Authority, which can be created using the resource type [`aws_acmpca_certificate_authority`](acmpca_certificate_authority.html).
```
Comment (breathingdust) [inline website/docs/cdktf/typescript/r/acm_certificate.html.markdown:47]: The language specific files are generated automatically from the main file in /website/docs/r so this change should be removed.

```suggestion
Private certificates are issued by an ACM Private Cerificate Authority, which can be created using the resource type [`aws_acmpca_certificate_authority`](acmpca_certificate_authority.html).
```

**Review by breathingdust — CHANGES_REQUESTED** [review]
Thanks for the contribution @heldersepu! Just need to remove the language specific file changes (as these are generated files) and this will be good to go.

**Comment by heldersepu** [comment]
@breathingdust suggestions applied...

**Comment by heldersepu** [comment]
@breathingdust suggestions applied...

If you have time can you please also review this othe PR of mine: https://github.com/hashicorp/terraform-provider-aws/pull/37800

**Comment by breathingdust** [comment]
LGTM 🚀 Thanks for the contribution @heldersepu!

**Pull request merged**